src/components/30-organisms/footer/README.md

```diff
@@ -0,0 +1,238 @@
+# Footer
+
+The big version of the footer. It is made for the use of several custom links and also to display social media buttons to the user.
+
+## Usage
+
+```bash
+npm install @axa-ch/footer
+```
+
+```js
+import '@axa-ch/footer';
+...
+
+const content = JSON.stringify([
```

Comment:
I believe, this is not possibile here to have a model. The footer must be indexed via google bot and needs its content inside DOM children.

Reply:
Please add aversion of purely DOM children *or* refactor this

Reply:
have a look at this SEO content as an inspiration example: https://github.com/axa-ch/patterns-library/blob/develop-v1/src/components/o-commercial-hero-cover/_preview.html